app/init/resources.php

```diff
@@ -843,3 +843,7 @@ function getDevice(string $root, string $connection = ''): Device
 }, ['request', 'project']);
 
 App::setResource('executor', fn () => new Executor(fn (string $projectId, string $deploymentId) => System::getEnv('_APP_EXECUTOR_HOST')));
+
+App::setResource('maxBatchSize', function () {
```

Comment:
We don't need a resource for holding a constant, over complicating.

Reply:
Also, we wanted to have different limit per different plans, so best if this is coming from the mock OSS plan and then overwritten on Cloud.

Reply:
Was overriding this resource to return the plan's value on cloud, have updated to use the plan directly to keep the pattern consistent